test_crates/unwind_function_missing/new/src/lib.rs

```diff
@@ -0,0 +1 @@
+pub extern "C" fn unwind_function_becomes_without_unwind() -> () {}
```

Comment:
In Rust, `-> ()` is implied when functions are not annotated as returning something else, so it's convention to omit it.
```suggestion
pub extern "C" fn unwind_function_becomes_without_unwind() {}
```